browser/ui/views/brave_news/brave_news_feed_item_view.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_BRAVE_NEWS_BRAVE_NEWS_FEED_ITEM_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_BRAVE_NEWS_BRAVE_NEWS_FEED_ITEM_VIEW_H_
+
+#include <vector>
+
+#include "brave/browser/brave_news/brave_news_tab_helper.h"
+#include "ui/views/controls/button/md_text_button.h"
+#include "ui/views/view.h"
+
+namespace content {
+class WebContents;
+}
+
+class BraveNewsFeedItemView : public views::View,
+                              public BraveNewsTabHelper::PageFeedsObserver {
+ public:
+  METADATA_HEADER(BraveNewsFeedItemView);
+
+  explicit BraveNewsFeedItemView(BraveNewsTabHelper::FeedDetails details,
```

Comment:
nit: I think we can remove explicit here.

I think the rule of thumb for `explicit` in upstream is
* When constructor has only one parameter.
* When constructor has multiple params but some default values for some of params so that we can call it with only one param.
* When a class is designed to be a simple wrapper or converted from other type, you can omit explicit intentionally. But rare.

```suggestion
  BraveNewsFeedItemView(BraveNewsTabHelper::FeedDetails details,
```

Reply:
Cool, that's really handy to know. Thanks @sangwoo108!